io/example/mets.xml

```diff
@@ -1,126 +1,350 @@
-            <mets:file ID="FILE_0004_FULLTEXT" MIMETYPE="text/xml">
-                <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000004.xml" /></mets:file>
-            <mets:file ID="FILE_0005_FULLTEXT" MIMETYPE="text/xml">
-                <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="https://github.com/OCR-D/spec/raw/master/io/example/00000005.xml" /></mets:file>
-        </mets:fileGrp>
-    </mets:fileSec>
+  <mets:metsHdr CREATEDATE="2017-11-30T16:18:26">
+    <mets:agent OTHERTYPE="SOFTWARE" ROLE="CREATOR" TYPE="OTHER">
+      <mets:name>DFG-Koordinierungsprojekt zur Weiterentwicklung von Verfahren der Optical Character Recognition (OCR-D)</mets:name>
+      <mets:note>OCR-D</mets:note>
+    </mets:agent>
+  </mets:metsHdr>
+    <!-- 
+         This section should contain all necessary information about the manuscript/book/...(Bitte durch korrekten Begriff ersetzen)
+         Mandatory values are:
+           ??? Gibt es eine Art ISBN-Nummer, die immer vorhanden sein sollte?
+           Welche Merkmale sollten hier auftauchen.
+           Wird das im Laufe des Workflows erweitert oder ist das nach Modul 1.A
+           fix?
+    -->
+  <mets:dmdSec ID="DMDLOG_0001">
+    <mets:mdWrap MDTYPE="MODS">
+      <mets:xmlData>
+        <mods:mods xmlns:mods="http://www.loc.gov/mods/v3">
+          <mods:location>
+            <mods:physicalLocation authority="marcorg" displayLabel="Staatsbibliothek zu Berlin - Preußischer Kulturbesitz, Berlin, Germany">DE-1</mods:physicalLocation>
+            <mods:shelfLocator>Gq 14350;Beil.3-1839</mods:shelfLocator>
+          </mods:location>
+          <mods:originInfo eventType="publication">
+            <mods:dateIssued encoding="iso8601" keyDate="yes">1839</mods:dateIssued>
+          </mods:originInfo>
+          <mods:originInfo eventType="digitization">
+            <mods:place>
+              <mods:placeTerm type="text">Berlin</mods:placeTerm>
+            </mods:place>
+            <mods:dateCaptured encoding="iso8601">2013</mods:dateCaptured>
+            <mods:publisher>Staatsbibliothek zu Berlin – Preußischer Kulturbesitz, Germany</mods:publisher>
+            <mods:edition>[Electronic ed.]</mods:edition>
+          </mods:originInfo>
+          <mods:classification authority="ZVDD">Rechtswissenschaft</mods:classification>
+          <mods:classification authority="ZVDD">Historische Drucke</mods:classification>
+          <mods:relatedItem type="host">
+            <mods:recordInfo>
+              <mods:recordIdentifier source="gbv-ppn">PPN767122410</mods:recordIdentifier>
+            </mods:recordInfo>
+          </mods:relatedItem>
+          <mods:recordInfo>
+            <mods:recordIdentifier source="gbv-ppn">PPN767137728</mods:recordIdentifier>
```

Comment:
Diese PPN ist der eindeutige Identifier für das Objekt INNERHALB einer Einrichtung. Um nationale Eindeutigkeit zu erreichen, müsste die PPN noch mit einem Präfix der ISIL-Datei versehen werden, so z.B. für diesen Titel:
DE-1a_PPN767122410

Reply:
ISIL: http://sigel.staatsbibliothek-berlin.de/en/startseite/

Reply:
Was bedeutet 'nationale Eindeutigkeit'? Gibt es dann nur einen Identifier für jedes Buch(?)?
Oder kann es mehrere geben, die dann jeweils eindeutig sind?

Reply:
Die PPN ist nur eindeutig je Einrichtung in D - die ISIL wiederum hat einen eindeutigen Identifier pro Einrichtung, sprich in Kombination müsste sich damit jeder Titel in D eindeutig referenzieren lassen.